wled00/json.cpp

```diff
@@ -536,6 +537,7 @@ void serializeInfo(JsonObject root)
   root[F("name")] = serverDescription;
   root[F("udpport")] = udpPort;
   root["live"] = (bool)realtimeMode;
+  root[F("mso")] = useMainSegmentOnly;  // using main segment only for live
```

Comment:
Better to do `root[F("liveseg")] = useMainSegmentOnly ? strip.getMainSegmentId() : -1;`, since that gives more flexibility for us to potentially decouple it from main segment in the future, and the API user saves a step because they do not need to look up `state.mainseg`.

Reply:
Ok. We will need to inform Lord-Grey (Hyperion) of the change.

Reply:
What does it mean? The LED Grouping Thread got closed. Does it mean Grouping will be possible or is it dead? I currently have 144LED/m stripes on 65“ (636LEDs) and have to use tmp2net. The Problemnis, i got a max of 38fps. I hoped of grouping to reach 60fps. I could solder a second wled with teo instances, but i hope gor the grouping ☺️

Reply:
Download and see. 😄

Reply:
i dont get it... i flashed the latest v13.2 "Toki" release, activated "main segment only" and nothing happend... still 38 fps... the fps counter in hyperHDR showed me a way higher transferrate (2-3x more fps), but the LEDs just get controlled by 38fps from the ESP32. I hoped that a grouping will increase the rate - but maybe i have to solder a second ESP32 for two instances.... or do you have a nother hint, how i can get more FPS??? i tried to split my 636 LEDs on two different pins - but FPS are still the same. It just increase, when i decrease the amount of LEDs.... my goal are 60FPS with 636LEDs... Ideas? (THX btw. for your nice & hard work!)

Reply:
@dimitrij2k Try increasing the `Target refresh rate` in the bottom of LED settings to 120. What grouping are you using, aka how many LEDs are transmitted?

Reply:
Hey Aircookie, i already played with the `Target refreshrate`, but it dont help. The problem is not only the live streaming with the fps. Even the "Effects" in WLED are running with only 38fps. 

My Setup: SK6812 RGBW LEDs are running with an ESP32-ETH01 (Cable) to a Asus AX86U Router.

Im using HyperHDR on my LG WebOS (installed on TV). As protocol only `tmp2net` is working correctly at this moment (because of the LED count). I use as `Max PacketSize 642` (for the LED Count + 6). Peak Performance is 38fps.

When i split my LEDs on two Pins (pin 2 = 315 LEDs, pin 3 = 321 LEDs) the 38fps are the same. **When i reduce the LED Count in WLED to 400LEDs, i get 60fps... below 400LEDs the fps rise, over 400LEDs the fps decrease.**

In HyperHDR i tried a direct LED mapping and Grouping of 10LEDs to 21 Zones (Top/Bottom) and 12 Zones (Left/Right)... but i never reach more then 38fps with my LED Count.

Thanks for your support!

Reply:
Your packet size is too small. And even the largest possible will not hold enough data for all LEDs. It will be fragmented.
At least as I see it from the description.